packages/inventory/src/Stocktake/ListView/ListView.tsx

```diff
@@ -7,13 +7,22 @@ import {
   TableProvider,
   createTableStore,
   useTranslation,
+  useTableStore,
 } from '@openmsupply-client/common';
 import { Toolbar } from './Toolbar';
 import { AppBarButtons } from './AppBarButtons';
-import { getStocktakeTranslator } from '../../utils';
+import { getStocktakeTranslator, isStocktakeDisabled } from '../../utils';
 import { StocktakeRowFragment } from '../api/operations.generated';
 import { useStocktakes } from '../api';
 
+const useDisableStocktakeRows = (rows?: StocktakeRowFragment[]) => {
+  const { setDisabledRows } = useTableStore();
+  useEffect(() => {
+    const disabledRows = rows?.filter(isStocktakeDisabled).map(({ id }) => id);
+    if (disabledRows) setDisabledRows(disabledRows);
+  }, [rows]);
+};
```

Comment:
disabling 'disabled' rows - i.e. locked and/or finalised.

Has the potential disadvantage that the row is grey/gray/grae - though it's a bit of feedback I suppose